hm_pyhelper/hardware_definitions.py

```diff
@@ -94,8 +108,12 @@ def is_raspberry_pi():
         'BUTTON': 23,
         'ECCOB': True,
         'TYPE': 'Full',
-        'CELLULAR': False
+        'CELLULAR': False,
+        'FCC_IDS': ['2AZDM-HNTLGT'],
```

Comment:
Need to update these - one sec will get the details

Reply:
Updated: https://github.com/NebraLtd/hm-pyhelper/pull/80/files#diff-2316c9dbb9d32df1562699aec0829c762a67684f74c5573db939f8ec1d5d7858R115